tests/everest/test_everserver.py

```diff
@@ -136,94 +123,8 @@ def test_configure_logger_failure(_, tmp_path, change_to_tmpdir):
 
 @patch("sys.argv", ["name", "--output-dir", "everest_output"])
 @patch("everest.detached.jobs.everserver._configure_loggers")
-@patch("requests.get")
-def test_status_running_complete(mocked_get, _, change_to_tmpdir):
-    def mocked_server(url, verify, auth, timeout, proxies):
```

Comment:
Is this test not relevant anymore?

Reply:
I might have been a bit :scissors: happy with this one I will put it back and update it.